File: Dataset/Leetcode/train/12/400.py
 class Solution:
    def XXX(self, num: int) -> str:
        nums = [1000, 900, 500, 400, 100, 90, 50, 40, 10, 9, 5, 4, 1]
        roma = ['M', 'CM', 'D', 'CD', 'C', 'XC',
                'L', 'XL', 'X', 'IX', 'V', 'IV', 'I']
        ans = []
        for i, x in enumerate(nums):
            if num >= x:
                ans.append(roma[i]*(num//x))
                num %= x
        return ''.join(ans)

File: Dataset/Leetcode/valid/121/400.java
 class Solution {
    public int XXX(int[] prices) {
        int ans=0;
        int max=prices[0];
        for(int  i=1;i<prices.length;++i){
            if(max>prices[i]){
                max=prices[i];
            }else{
                ans=Math.max(ans,prices[i]-max);
            }
        }
        return ans;
    }
}

